packages/components/psammead-media-player/src/MediaMessage/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { string } from 'prop-types';
+import styled from 'styled-components';
+import { C_WHITE } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING_DBL, GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { GEL_LONG_PRIMER } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { GEL_GROUP_2_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import Image from '@bbc/psammead-image';
+
+const NOJS_BACKGROUND_COLOUR = 'rgba(34, 34, 34, 0.75)';
+
+const StyledWrapper = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+`;
+
+const MessageWrapper = styled.div`
+  ${({ service }) => getSansRegular(service)}
+  ${GEL_LONG_PRIMER};
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  border: 0.0625rem solid transparent;
+  color: ${C_WHITE};
+  background-color: ${NOJS_BACKGROUND_COLOUR};
+  @media screen and (-ms-high-contrast: active) {
+    background-color: transparent;
+  }
+`;
+
+const Message = styled.strong`
+  display: block;
+  font-weight: normal;
+  bottom: 0;
+  position: absolute;
+  padding: ${GEL_SPACING};
+  border-bottom: 0.0625rem solid transparent;
+  @media screen and (-ms-high-contrast: active) {
```

Comment:
Nice - I think this is the first time I've seen a PR catch that before it's found in a11y testing 😄